production-deployment-using-tiup.md

```diff
@@ -0,0 +1,1927 @@
+---
```

Comment:
2 files need to be deleted. And you can check globally whether there are other `<br>` to be updated.